meta-webserver/recipes-webadmin/cockpit/cockpit_217.bb

```diff
@@ -0,0 +1,178 @@
+SUMMARY = "Admin interface for Linux machines"
+DESCRIPTION = "Cockpit makes it easy to administer your GNU/Linux servers via a web browser"
+
+LICENSE = "LGPLv2+"
```

Comment:
is this 2+ or LGPL-2.1?

Reply:
Good point. License is "LGPLv2.1". I will correct it.

Reply:
Is now corrected